podcasts/ThemeColor.swift

```diff
@@ -4,7 +4,7 @@ import PocketCastsUtils
 struct ThemeColor {
     private static let primaryUi01Light = UIColor(hex: "#FFFFFF")
     private static let primaryUi01Dark = UIColor(hex: "#292B2E")
-    private static let primaryUi01ExtraDark = UIColor(hex: "#222427")
+    private static let primaryUi01ExtraDark = UIColor(hex: "#000000")
```

Comment:
We can't edit this file, as stated by this line it is auto-generated: https://github.com/Automattic/pocket-casts-ios/blob/trunk/podcasts/ThemeColor.swift#L3

What we can do is tweak the places that use those colors. You can compare all available colors here: https://podcast.emily.tools/themes/?pc_validation

Reply:
@leandroalonso Looking at the [theme colors](https://podcast.emily.tools/themes/?pc_validation) link, it seems that the ExtraDark theme is displaying as expected.

It may be worth revisiting the color palette for the Extra Dark theme, as it seems to be a slightly darker palette than the Default Dark theme. The Theme selection previews for Extra Dark depict a monochrome palette without any accent colors (i.e., icons as #50D0F1). 

For now, I'll submit this PR with only the Search bar padding for the Choose Podcasts screen and leave the palette for the Extra Dark theme untouched.

![SearchBarPadding-side](https://github.com/Automattic/pocket-casts-ios/assets/81433983/038d7839-415d-4391-91af-215559771a4a)